File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/save-training-event-stakeholder_item.ts


import { ISaveTrainingEventStakeholder_Item } from './isave-training-event-stakeholder_item';

export class SaveTrainingEventStakeholder_Item implements ISaveTrainingEventStakeholder_Item {
  
	public TrainingEventID?: number;
	public AppUserID?: number;
	public ModifiedByAppUserID?: number;
	public ModifiedDate: Date = new Date(0);
  
}

File: GTTS/INL.TrainingService.Models/SaveTrainingEventStakeholder_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.TrainingService.Models
{
    public class SaveTrainingEventStakeholder_Item : ISaveTrainingEventStakeholder_Item
    { 
        public Int64? TrainingEventID { get; set; }
        public int? AppUserID { get; set; }
        public int? ModifiedByAppUserID { get; set; }
        public DateTime ModifiedDate { get; set; }
    }
}
